packages/eds-core-react/src/components/Popover/PopoverHeader.tsx

```diff
@@ -0,0 +1,35 @@
+import { forwardRef, HTMLAttributes } from 'react'
+import styled, { css } from 'styled-components'
+import { Divider } from '../Divider'
+
+export type PopoverHeaderProps = HTMLAttributes<HTMLDivElement>
+
+const StyledPopoverHeader = styled.div(({ theme }) => {
+  return css`
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: calc(${theme.spacings.top} - ${theme.entities.title.spacings.top})
```

Comment:
Maybe just specify it to be `12px` in `theme.entities.title.spacings.top` for clarity. This seems to be an exception to normal spacing rules.  Would also verify with designers if this is intended as it seems weird with only this being 16px while the rest is 16px 😅

Reply:
Just try to be sophisticated 😅
I will talk to the designers 👍